assembly/hdp2.6.4/build/var/lib/ambari-server/resources/common-services/SPLICEMACHINE/2.5.1/service_advisor.py

```diff
@@ -0,0 +1,240 @@
+
+#    siteName = "hbase-site"
+#    method = self.validateHBaseSiteConfigurations
+#    resultItems = self.validateConfigurationsForSite(configurations, recommendedDefaults, services, hosts, siteName, method)
+#    items.extend(resultItems)
+
+  def validateCoreSiteConfigurations(self, properties, recommendedDefaults, configurations, services, hosts):
+    print "validateCoreSiteConfigurations"
+    core_site = properties
+    validationItems = []
+    for property, desired_value in self.getCoreSiteDesiredValues().iteritems():
+      if property not in core_site or core_site[property] != desired_value:
+        message = "Splice Machine requires this property to be set to the recommended value of " + desired_value
+        validationItems.append({"config-name": property, "item": self.getWarnItem(message)})
+    return self.toConfigurationValidationProblems(validationItems, "core-site")
+
+  def validateHDFSSiteConfigurations(self, properties, recommendedDefaults, configurations, services, hosts):
+      print "validateHDFSSiteConfigurations"
+      hdfs_site = properties
+      validationItems = []
+      for property, desired_value in self.getHDFSSiteDesiredValues().iteritems():
+          if property not in hdfs_site or hdfs_site[property] != desired_value:
+              message = "Splice Machine requires this property to be set to the recommended value of " + desired_value
+              validationItems.append({"config-name": property, "item": self.getWarnItem(message)})
+      return self.toConfigurationValidationProblems(validationItems, "hdfs-site")
+
+  def validateHBaseSiteConfigurations(self, properties, recommendedDefaults, configurations, services, hosts):
+      print "validateHBaseSiteConfigurations"
+      hbase_site = properties
+      validationItems = []
+      for property, desired_value in self.getHBaseSiteDesiredValues().iteritems():
+          print "->" + property + ":" + desired_value + ":" + hbase_site[property]
+          if property not in hbase_site or hbase_site[property] != desired_value:
+              message = "Splice Machine requires this property to be set to the recommended value of " + desired_value
+              validationItems.append({"config-name": property, "item": self.getWarnItem(message)})
+      return self.toConfigurationValidationProblems(validationItems, "hbase-site")
+
+  def getCoreSiteDesiredValues(self):
+    core_site_desired_values = {
+        "ipc.server.listen.queue.size" : "3300"
+    }
+    return core_site_desired_values
+
+  def getHDFSSiteDesiredValues(self):
+    hdfs_site_desired_values = {
+        "dfs.datanode.handler.count" : "20",
+        "dfs.client.read.shortcircuit.buffer.size" : "131072",
+
+    }
+    return hdfs_site_desired_values
+
+  def getHBaseSiteDesiredValues(self):
+    hbase_site_desired_values = {
+        "hbase.coprocessor.master.classes" : "org.apache.ranger.authorization.hbase.RangerAuthorizationCoprocessor,com.splicemachine.hbase.SpliceMasterObserver",
```

Comment:
DB-7222 removed `org.apache.ranger.authorization.hbase.RangerAuthorizationCoprocessor`, this PR need to remove this, too.